src/lib-components/BlockEvent.vue

```diff
@@ -0,0 +1,189 @@
+<script>
+import format from 'date-fns/format'
+import SmartLink from '@/lib-components/SmartLink'
+
+// Utility functions
+import getSectionName from '@/utils/getSectionName'
+import formatTimes from '@/utils/formatEventTimes'
+import formatDates from '@/utils/formatEventDates'
+
+export default {
+  name: 'BlockEvent',
+  components: { SmartLink },
+  props: {
+    title: {
+      type: String,
+      default: '',
+    },
+    category: {
+      type: String,
+      default: '',
+    },
+    startDate: {
+      type: String,
+      default: '',
+    },
+    endDate: {
+      type: String,
+      default: '',
+    },
+    prompt: {
+      type: String,
+      default: 'View',
+    },
+    to: {
+      type: String,
+      default: '',
+    },
+    sectionHandle: {
+      type: String,
+      default: '',
+    },
+  },
+  computed: {
+    classes() {
+      return ['block-event', `button color-${this.sectionName}`]
+    },
+    sectionName() {
+      return getSectionName(this.to)
+    },
+    parsedDate() {
+      return formatDates(this.startDate, this.endDate)
+    },
+    parsedSingledDate() {
+      return format(new Date(this.startDate), 'MMMM d, Y')
+    },
+    parsedTime() {
+      if (this.startDate && this.sectionHandle === 'event')
+        return formatTimes(this.startDate, this.endDate)
+
+      return ''
+    },
+  },
+}
+</script>
+
+<template>
+  <div :class="classes">
+    <div class="molecule row" />
+
+    <div class="text row">
+      <span class="category">{{ category }}</span>
+      <h2 class="title" v-html="title" />
```

Comment:
use v-text https://vuejs.org/api/built-in-directives.html#v-text or mustache template syntax to pass title to h2 element